### DISCUSSION TITLE
独立审核及站点列表库，征求技术方案和建议

### DISCUSSION THREAD

**Discussion by HAHH9527** [travellings-link/travellings#1181] [寻求帮助]
目前开往的前端页和站点列表高度耦合，UI改版和站点新增/撤下都会互相影响，现向各位征求可行的技术方案和建议。

**Comment by HAHH9527** [comment]
抛砖引玉，我在此先提出一个我初步设想的方案：
站点列表改为维护csv文件，方便审核人员编辑。
编写脚本，以csv为基础自动生成站点列表json供js读取。
优点：csv文件可读性强 方便审核人员编辑
缺点：需要额外编写csvToJson脚本

**Comment by linlinzzo** [comment]
Agree.
